breakpoints.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable vue/sort-keys */
```

Comment:
What is the reason to disable this rule?

Reply:
Because otherwise it wants to sort the key alphabetically when it makes more sense to sort them by size. One of the examples why I really dislike such opinionated eslint rules, but this PR isn't about that.

Reply:
Yeah, I think that makes a lot of sense